app/javascript/toggle_action.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener("turbolinks:load", function() {
+  $(".btn-show-details").on("click", function() {
+    var $button = $(this);
+
+    if ($button.attr("aria-expanded") === "false") {
+      $button.text("Show Details");
+    } else {
+      $button.text("Hide Details");
+    }
+  });
+
+  $(".order-cancellation-form .order_cancellation_reason").on("input", function() {
+    var cancellationReason = $(this).val().trim();
+    var cancelButton = $(this).closest(".order-cancellation-form").find(".cancel-button");
```

Comment:
```suggestion
    var cancelButton = $(this).closest(".order-cancellation-form").find(".js-cancel-button");
```
nhớ có comment vụ class js vs css tách biệt rồi đúng không nhỉ ?

Reply:
Dạ oki a